cmd/network/verification.go

```diff
@@ -332,6 +336,9 @@ func (e *EgressVerification) generateAWSValidateEgressInput(ctx context.Context,
 	}
 	input.AWS.SecurityGroupId = sgId
 
+	// Forward the timeout
+	input.Timeout = e.EgressTimeout
```

Comment:
The `osd-network-verifier` default timeout is actually 2s as well.
For the command line tool at least.
When called as a library, as `osdctl` does, there is indeed no default timeout. I wonder what happens in that case if none is specified. Checking.

Reply:
Sorry, I was quite busy last week.
Anyway, @AlexVulaj , here is the result of the test: if `input.Timeout` is not set then the network verifier will wait indefinitely. This could be what we want... at least when `osdctl network verify-egress` is called by a real user; however the command is also called from some shell script (`cluster_has_gone_missing`) in our SOPs, this script may therefore wait indefinitely in the unlikely scenario egresses never succeed or fail.

Reply:
Actually increasing the timeout may not be that of a great idea.
Indeed we don't know what is the real timeout used by pods / operators when they call those endpoints.
Increasing the timeout on `osdctl` may hide issues we could encounter with those pods / operators